source/docs/api-reference/api-usage/actuator-limits.rst

```diff
@@ -3,6 +3,8 @@ Actuator Limits
 
 CTR Electronics actuators, such as the TalonFX, support various kinds of hardware and software limits.
 
+.. note:: The TalonFX + Kraken x60 does not support hardware limit switches. Instead, a CANcoder can be used a remote limit switch.
```

Comment:
The control limits API should be the first suggestion here, followed by "...or a CANcoder can be used as a remote limit switch."